mapit_global/management/commands/mapit_global_import.py

```diff
@@ -31,7 +31,7 @@
 from django.utils.six.moves import urllib
 from django.utils.encoding import smart_str, smart_text
 
-import shapely
+import shapely.wkb
```

Comment:
This line in `bin/generate_kml.py` activates shapely.wkb: `from shapely.geometry import Polygon`.